values.yaml

```diff
@@ -94,15 +94,15 @@ images:
     pullPolicy: IfNotPresent
   redis:
     repository: quay.io/astronomer/ap-redis
-    tag: 0.11.0
+    tag: 6.2.1
     pullPolicy: IfNotPresent
   pgbouncer:
     repository: quay.io/astronomer/ap-pgbouncer
-    tag: 0.11.0
+    tag: 1.8.1
     pullPolicy: IfNotPresent
   pgbouncerExporter:
     repository: quay.io/astronomer/ap-pgbouncer-exporter
-    tag: 0.11.0
+    tag: 0.9.2
```

Comment:
Are the changes between these versions non-breaking?

Reply:
I'm honestly not sure. I'll deploy this version of the chart to dev so we can see.